public/app/plugins/panel/singlestat/editor.html

```diff
@@ -29,7 +29,7 @@ <h5 class="section-heading">Value</h5>
         <input type="text" class="gf-form-input width-12" ng-model="ctrl.panel.prefix" ng-change="ctrl.render()" ng-model-onblur>
         <label class="gf-form-label width-6">Font size</label>
         <div class="gf-form-select-wrapper">
-          <select class="gf-form-input" ng-model="ctrl.panel.prefixFontSize" ng-options="f for f in ctrl.fontSizes" ng-change="ctrl.render()"></select>
+          <select class="gf-form-input" ng-model="ctrl.panel.prefixFontSize" ng-options="f for f in ctrl.fontSizes" ng-change="ctrl.render()" ng-disabled="ctrl.panel.gauge.show"></select>
```

Comment:
I think we should use a function to enable/disable this feature instead of having to many decisions in the view.
So instead of `ng-disabled="ctrl.panel.gauge.show"` we should have something like `ng-show="ctrl.showFontsizes()"`

We will start migrating the frontend to react and having functions decide if thing should show up or not makes it easier.

Reply:
Just curious, why move from Angular to React? Wouldn't it be easier to instead upgrade to Angular 2? Is there something Angular is not doing well?

Reply:
Upgrading to angular two is as big change as migrating to react.

Reply:
Also, I've pushed a new commit. Is that what you were expecting?